wallet-new/server/Main.hs

```diff
@@ -152,11 +153,24 @@ actionWithNewWallet pm txpConfig sscParams nodeParams ntpConfig params =
         -> (Diffusion Kernel.Mode.WalletMode -> Kernel.Mode.WalletMode ())
     runNodeWithInit w nr = runNode pm txpConfig nr (plugins w)
 
-    -- TODO: Don't know if we need any of the other plugins that are used
-    -- in the legacy wallet (see 'actionWithWallet').
+    -- FIXME: Do we need the monitoring API ?
```

Comment:
I think we might, but I can pick this up while you on holiday, @KtorZ 😉

Reply:
Apparently, ops don't use it. Maybe exchanges do?